Add custom 404 page (#692)
SwedbankPay/developer.swedbankpay.com: 404.md

---
permalink: /404.html
---

<!-- markdownlint-disable MD026 -->
# Uh oh:heavy_exclamation_mark: Seems like you have taken a wrong turn somewhere :construction:
<!-- markdownlint-enable MD026 -->

This page seems to try to hide from you. :dragon_face:

:circus_tent:

> Perhaps looking in the searchbar or clicking around in the menus will help?

Reviewer: ```suggestion
Perhaps attempting a search or clicking around in the menus will help?
```

permalink: /404.html
---

<!-- markdownlint-disable MD026 -->
# Uh oh:heavy_exclamation_mark: Seems like you have taken a wrong turn somewhere :construction:
<!-- markdownlint-enable MD026 -->

> This page seems to try to hide from you. :dragon_face:

Reviewer 2: ```suggestion
This page seems to play hide and seek with you. :dragon_face:
```
Just playing around :)
